From f1e1ea6513c6d7a4a21a970c33659ffd63149b5f Mon Sep 17 00:00:00 2001
From: Roman Lebedev <lebedev.ri@gmail.com>
Date: Fri, 23 Aug 2024 02:42:18 +0300
Subject: [PATCH 6/9] Disable some more tests for gecode solver

These fail with
```
E     but got
E
E     !Error
E     message: 'GecodeSolverInstance::processFlatZinc: Error: Unbounded variable: X_INTRODUCED_80_,
E       rerun with --allow-unbounded-vars to add arbitrary bounds.
E
E       '
E     type: MiniZincError

```

diff --git a/libminizinc/tests/spec/examples/cutstock.mzn b/libminizinc/tests/spec/examples/cutstock.mzn
index 447e80da1..13c1a536e 100644
--- a/libminizinc/tests/spec/examples/cutstock.mzn
+++ b/libminizinc/tests/spec/examples/cutstock.mzn
@@ -1,5 +1,6 @@
 /***
 !Test
+solvers: []
 expected:
 - !Result
   solution: !Solution
diff --git a/libminizinc/tests/spec/examples/product_lp.mzn b/libminizinc/tests/spec/examples/product_lp.mzn
index a7d2365da..e4c1eea4f 100644
--- a/libminizinc/tests/spec/examples/product_lp.mzn
+++ b/libminizinc/tests/spec/examples/product_lp.mzn
@@ -1,5 +1,6 @@
 /***
 !Test
+solvers: []
 expected:
 - !Result
   solution: !Solution
diff --git a/libminizinc/tests/spec/examples/warehouses.mzn b/libminizinc/tests/spec/examples/warehouses.mzn
index 7ab5e145a..5ca21f82b 100644
--- a/libminizinc/tests/spec/examples/warehouses.mzn
+++ b/libminizinc/tests/spec/examples/warehouses.mzn
@@ -1,5 +1,6 @@
 /***
 !Test
+solvers: []
 expected:
 - !Result
   solution: !Solution
-- 
2.45.2

